src/Workspaces/Core/Portable/WorkspacesResources.resx

```diff
@@ -565,4 +565,7 @@
   <data name="Sync_namespace_to_folder_structure" xml:space="preserve">
     <value>Sync namespace to folder structure</value>
   </data>
+  <data name="The_contents_of_a_SourceGeneratedDocument_may_not_be_changed" xml:space="preserve">
+    <value>The contents of a SourceGeneratedDocument may not be changed.</value>
```

Comment:
`The contents of a 'SourceGeneratedDocument' may not be changed.`

Reply:
or `The contents of a '{0}' may not be changed.`

Reply:
Normally I'm using the quoted form only if that string is user-supplied which isn't really the case here; I thought about parameterizing but then it's funky for the localizer if they need to deal with something like a/an in English.

Reply:
(so won't change this unless there's strong objections)